src/ChainSafe.Gaming/Web3/Core/Chains/ChainManager.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ChainSafe.Gaming.Web3.Environment;
+
+namespace ChainSafe.Gaming.Web3.Core.Chains
+{
+    public class ChainManager : IChainManager
```

Comment:
Can we add the functionality to add chains to the list like IChainManager.AddChain(IChainConfig chainConfig)

Reply:
Added `AddChainConfig` method to `IChainManager` ✅